core/src/events.rs

```diff
@@ -32,6 +32,9 @@ pub enum PlayerEvent {
     TextInput {
         codepoint: char,
     },
+    TextControl {
```

Comment:
Nit: Would a (maybe one-line) comment describing what the event means be helpful? I do not understand this part of the codebase, however, so that might be superfluous.